CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# How to contribute
+
+PROJECT is Apache 2.0 licensed and accepts contributions via GitHub pull
```

Comment:
Not sure if we need this file or not? architect doesn't check for it. If we do add we should fix the project name.

Reply:
I think it makes sense since this is one of our repos, that actually got/get contributions, WDYT?  I'll change the project name

Reply:
👍 Good call to add it. Makes sense to have on repos where we get contributions.